src/Cart.Service/Cart.Service/Program.Services.cs

```diff
@@ -17,6 +22,16 @@ public static WebApplicationBuilder RegisterAppServices(this WebApplicationBuild
         builder.Services.AddEndpointsApiExplorer();
         builder.Services.AddAppSwagger();
 
+        builder.Services.Configure<MessageBusOptions>(builder.Configuration.GetSection(MessageBusOptions.Position));
+        builder.Services.AddAzureClientsCore(false);
+        builder.Services.AddSingleton(sp =>

```

Comment:
I'd suggest using Azure SDK for registration, see:
[AddAzureClients](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/messaging.servicebus-readme?view=azure-dotnet#registering-with-aspnet-core-dependency-injection)

Reply:
Updated